config/rollup/rollup.config.js

```diff
@@ -173,6 +169,27 @@ function configure(pkg, env, target) {
       },
     }
   }
+
```

Comment:
this is identical to what it was before but now just handles es6 only, no cjs output here